packaging/zip/Eluant.dll.config

```diff
@@ -0,0 +1,4 @@
+<configuration>
+  <dllmap os="linux" dll="lua51.dll" target="liblua5.1.so.0" />
```

Comment:
This will break compatibility with most non-debian/ubuntu linux distros.  See https://github.com/OpenRA/OpenRA/pull/5460, https://github.com/OpenRA/OpenRA/pull/6782, https://github.com/OpenRA/OpenRA/pull/7652, https://github.com/OpenRA/OpenRA/pull/12441, https://github.com/OpenRA/OpenRA/pull/12631.

You will need to configure this file on the fly as part of a portable-package-only launch script if you want this to work portably across linux distros.